### ISSUE TITLE
How to create a new mix project

### ISSUE THREAD

**Issue by bbtdev** [KronicDeth/intellij-elixir#1133]
Hello,

In the documentation it is explained how to create a basic (non-mix) Elixir project, but there is no explanation if there is a basic mix Elixir project setup. Mix must be run manually from the terminal?

If this is the case, maybe it is possible to mention this in the documentation.

Thank you

**Comment by KronicDeth** [comment]
Yeah, you still need to run mix from the commandline to setup a project.  There are just too many options for `mix new` and `mix phx.new` to make it make sense to integrate it into the UI.

**Comment by bbtdev** [comment]
@KronicDeth I understand, thank you for your response
